### ISSUE TITLE
Using OS.set_window_size() & OS.set_window_resizable(false) in conjunction with moving the game window

### ISSUE THREAD

**Issue by razvanc-r** [godotengine/godot#6572]
**Operating system or device - Godot version:**
Ubuntu 16.04 - Godot v2.1.stable.official

**Issue description** (what happened, and what was expected):
After using

``` python
OS.set_window_size(Vector2(...))
OS.set_window_resizable(false)
```

in a script if you move the game window the size jumps to what is set in the `Project Settings`

**Steps to reproduce:**
Run attached minimal example, try moving the wind & see script in attached minimal example.

**Link to minimal example project** (optional but very welcome):
[debug.zip](https://github.com/godotengine/godot/files/483846/debug.zip)

**Comment by marcelofg55** [comment]
Hi, I've found this problem also while using the resizable flag set to false, this patch should fix it, I have tested it on Ubuntu with Godot v2.2-legacy, could you try it out and confirm it fixes the issue?
Thanks

[godot_x11_resize.patch.zip](https://github.com/godotengine/godot/files/730926/godot_x11_resize.patch.zip)

**Comment by bojidar-bg** [comment]
@marcelofg55 Do you think you can make a PR against master?

**Comment by marcelofg55** [comment]
@bojidar-bg I was just trying to compile master but it's giving me some errors, I guess reduz is probably working on some new stuff, I'll submit a PR after I can compile & test it.

**Comment by marcelofg55** [comment]
@bojidar-bg Just submitted the PR: #7683

**Comment by Noshyaar** [comment]
Closing as fixed by #7683
